test/e2e/tests/multitenancy_helpers.py

```
@@ -1337,35 +1338,87 @@ def search_api_keys_at(
+        ])
+    curl_args.extend(["-w", "\\nHTTP_CODE:%{http_code}", url])
+
+    pod_name = f"mt-curl-{os.getpid()}-{uuid.uuid4().hex[:6]}"
+    namespace = os.environ.get("E2E_CURL_POD_NAMESPACE", E2E_CURL_POD_NAMESPACE)
+    cmd = [
+        "kubectl", "run", pod_name,
+        "--rm", "-i", "--restart=Never",
+        "--image=curlimages/curl:latest",
+        "-n", namespace,
+        "--", "curl",
+    ] + curl_args
```

Ruff (0.16.1)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)